iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractTapChanger.java

```diff
@@ -41,12 +45,12 @@
 
     protected AbstractTapChanger(Ref<? extends VariantManagerHolder> network, H parent,
                                  int lowTapPosition, List<S> steps, TerminalExt regulationTerminal,
-                                 int tapPosition, boolean regulating, double targetDeadband) {
+                                 int tapPosition, boolean regulating, double targetDeadband, String type) {
```

Comment:
Was it really necessary to add the type here? Depending on which java class you're in you basically know the type of equipment considered, or am I wrong?